Makefile

```diff
@@ -123,22 +123,23 @@ clean:
 	$(ECHO_V)rm -rf examples/keyvalue/kv/ .bin
 
 .PHONY: examples
-examples: .bin/thriftrw .bin/thriftrw-plugin-yarpc
-	@$(call label,Installing thriftrw and YARPC plugins)
-	@echo
-	$(ECHO_V)which thriftrw-plugin-fx >/dev/null || go install ./modules/rpc/thriftrw-plugin-fx
+examples: .bin/thriftrw .bin/thriftrw-plugin-yarpc .bin/thriftrw-plugin-fx
 	@$(call label,Generating example RPC bindings)
 	@echo
 	$(ECHO_V)PATH=$(shell pwd)/.bin:$$PATH $(MAKE) -C examples/keyvalue kv/types.go ECHO_V=$(ECHO_V)
 
-.bin/thriftrw: vendor
+.bin/thriftrw: vendor ./vendor/go.uber.org/thriftrw/*.go
 	$(ECHO_V)mkdir -p .bin
 	$(ECHO_V)./.build/build_vendored.sh .bin go.uber.org/thriftrw
 
-.bin/thriftrw-plugin-yarpc: vendor
+.bin/thriftrw-plugin-yarpc: vendor ./vendor/go.uber.org/yarpc/encoding/thrift/thriftrw-plugin-yarpc/*.go
 	$(ECHO_V)mkdir -p .bin
 	$(ECHO_V)./.build/build_vendored.sh .bin go.uber.org/yarpc/encoding/thrift/thriftrw-plugin-yarpc
 
+.bin/thriftrw-plugin-fx: vendor ./modules/rpc/thriftrw-plugin-fx/*.go
+	$(ECHO_V)mkdir -p .bin
```

Comment:
Can you re-add the `call label...` stuff so we know what's happening while this is silently building?